libs/designsystem/header-actions/src/header-actions.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="bounding-box" #boundingBox>
```

Comment:
Maybe this could just be the host element itself?

Reply:
Nice catch! I actually did just that in the follow-up PR in https://github.com/kirbydesign/designsystem/pull/2920 (which is not yet done)